src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -1,34 +1,37 @@
 @mixin checkbox-icon() {
     border: 2px solid $ez-color-secondary;
-    padding: .2rem;
+    padding: 4px;
     margin: 0;
-    line-height: 1;
-    border-radius: 5px;
+    line-height: .5;
+    border-radius: .25rem;
     position: relative;
     z-index: 1;
     cursor: pointer;
 
     &:before {
         content: "";
         position: absolute;
-        width: 2.3rem;
-        height: 2.3rem;
+        width: 2rem;
+        height: 2rem;
         top: 50%;
         left: 0;
         background-color: $ez-color-secondary;
         z-index: -1;
-        border-radius: 5px;
+        border-radius: .25rem;
         transform: translate(calc(100% + .3rem), -50%);
-        transition: all .2s $ez-admin-transition;
+        transition: all .2s ease-in;
```

Comment:
Q: shouldn't we have one transition style across whole application?

Reply:
I agree, we should have one type for consistency.

Reply:
I am not totally sure about if we should apply the same kind of animation and timing for all cases. We have to be consistent, but differentiate cases based on the experience we would like to define.

Reply:
I believe that for now, we should use the same transition timing function everywhere and do some research what works better in each case. Transitions should be meaningful (in some way). For me, using standard easing functions makes UI less responsive.